application/src/main/resources/thingsboard.yml

```diff
@@ -1206,6 +1218,9 @@ vc:
     io_pool_size: "${TB_VC_GIT_POOL_SIZE:3}"
     repositories-folder: "${TB_VC_GIT_REPOSITORIES_FOLDER:${java.io.tmpdir}/repositories}"
 
+notification_system:
+  thread_pool_size: "${TB_NOTIFICATION_SYSTEM_THREAD_POOL_SIZE:30}"
```

Comment:
Wow, why do you need 30? I know that it shrinks if not required, but just curious.

Reply:
For most thread pools we have sizes from 40 to 50, just followed the trend :) Let's change the size if you want